src/modules/categories/components/CategoryListItem.js

```diff
@@ -73,9 +75,10 @@ class CategoryListItem extends React.Component<PropsType> {
   }
 
   renderTitle (): React.Node {
-    const { query, theme } = this.props
-    return <CategoryCaption search={query || ''} theme={theme}>
-      <CategoryTitle theme={theme}>{this.props.category.title}</CategoryTitle>
+    const { query, theme, category } = this.props
+    return <CategoryCaption theme={theme}>
+      <CategoryTitle theme={theme} textToHighlight={category.title} sanitize={normalize}
+                     searchWords={query ? [query] : []} highlightStyle={{ backgroundColor: 'yellow' }} />
```

Comment:
I think we use here the integreat color or make this configurable for the dark theme.